e2e-tests/development-runtime/gatsby-config.js

```diff
@@ -1,3 +1,5 @@
+require(`isomorphic-fetch`)
```

Comment:
Nice! I would strongly suggest adding some context on why this is here (links to issues, PRs) and note it can be removed once new `source-map` will be released and react-hot-loader bumps to use it.